hudi-common/src/main/java/org/apache/hudi/common/util/HoodieRecordUtils.java

```diff
@@ -73,33 +74,30 @@ public static HoodieRecordMerger loadRecordMerger(String mergerClass) {
    * Instantiate a given class with a record merge.
    */
   public static HoodieRecordMerger createRecordMerger(String basePath, EngineType engineType,
-      List<String> mergerClassList, String recordMergerStrategy) {
```

Comment:
Can we write a test case for it in `TestHoodieRecordUtils`

Reply:
done